### PR TITLE
Add python-box to conda-forge

### PR DESCRIPTION
**Pull request by jan-janssen**

### CODE REVIEW

File: recipes/python-box/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+    - pytest-runner
+    - pyyaml
```
Comment (CJ-Wright) [inline recipes/python-box/meta.yaml]: Are all of these build requirements?

Comment (jan-janssen) [reply recipes/python-box/meta.yaml]: pyyaml - can be removed, but pytest-runner is required as the tests are executed after the build process to validate the installation. @CJ-Wright should I also add you to the list of maintainers? https://github.com/cdgriffith/Box/pull/50#issuecomment-377513167

**Comment by jan-janssen** [comment]
@CJ-Wright I am sorry I meant to put @cdgriffith - the updated version is included in https://github.com/conda-forge/staged-recipes/pull/5508/commits/e71ba53b355928cb1f55e53fbc7986eb9eff8096

File: recipes/python-box/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+
+test:
+  imports:
+    - box
+
+about:
+  home: https://github.com/cdgriffith/Box
+  license: MIT
+  license_family: MIT
```
Comment (CJ-Wright) [inline recipes/python-box/meta.yaml]: would it be possible to include the license file?

**Comment by CJ-Wright** [comment]
Thank you very much!

**Pull request merged**